lighthouse-core/gather/gatherers/offline.js

```diff
@@ -34,4 +36,8 @@ class Offline extends Gatherer {
   }
 }
 
+function cleanHash(url) {
+  return url.href.removeURLFragment();
```

Comment:
maybe do the `new URL()` part inside here too? Can also do `url.hash = ''; return url.href;`